rotkehlchen/accounting/mixins/event.py

```diff
@@ -78,3 +78,12 @@ def process(
     @abstractmethod
     def serialize(self) -> Dict[str, Any]:
         ...
+
+    @abstractmethod
+    def serialize_for_import(self) -> Dict[str, Any]:
```

Comment:
I see that you implement this in all subclasses. But why?

Also perhaps namecould be adjust. `serialize_for_debug_import` to specify what its difference from `serialize` is.

All implementations are identical? Why not just implement it here?

Imlementation is the same for everything

```python

        data = self.serialize()
        data['accounting_event_type'] = self.get_accounting_event_type().serialize()
 ```